src/pipelinedefinition.cpp

```diff
@@ -675,12 +676,32 @@ Status PipelineDefinition::validateNodes(ModelManager& manager) {
             return StatusCode::PIPELINE_NODE_NAME_DUPLICATE;
         }
 
+        if (node.demultiplyCount) {
+            isDemultiplexerUsed = true;
+        }
+
+        if (manager.modelExists(node.modelName)) {
+            auto modelInstance = manager.findModelByName(node.modelName)->getDefaultModelInstance();
+            if (modelInstance) {
+                for (auto& [inputName, tensorInfo] : modelInstance->getInputsInfo()) {
+                    if (!tensorInfo->getShape().empty() && tensorInfo->getShape()[0] >= 2) {
+                        isBsGreaterThan1Used = true;
```

Comment:
```suggestion
                        isBsGreaterThan1Used = true;
                        break;
```

Reply:
You could as well use std::any_of if you want as well.
https://en.cppreference.com/w/cpp/algorithm/all_any_none_of